community/zsh/build

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh -e
+
+patch -p1 < remove_bin_duplicate.patch
+
+for _fpath in AIX BSD Cygwin Darwin Debian Mandriva openSUSE Redhat Solaris; do
+    rm -rf Completion/$_fpath
```

Comment:
Quote the argument with the variable.

Reply:
That's not necessary in this case, but sure.